### PR TITLE
Add num64_ prefix to comparison ops

### PR DESCRIPTION
**Pull request by sanket1729**
Followup to #27

### CODE REVIEW

File: doc/extension_spec.md [hunk 1]
```diff
@@ -44,13 +44,13 @@ To use these with miniscript fragments, we can use them inside comparison extens
...
 Name                                    | Script
 ---                                     | ---
-num_eq(NumExpr_X,NumExpr_Y)             | `[NumExpr_X] [NumExpr_Y] EQUAL64`
-le(NumExpr_X,NumExpr_Y)                 | `[NumExpr_X] [NumExpr_Y] LESSTHAN64`
-ge(NumExpr_X,NumExpr_Y)                 | `[NumExpr_X] [NumExpr_Y] GREATERTHAN64`
-leq(NumExpr_X,NumExpr_Y)                | `[NumExpr_X] [NumExpr_Y] LESSTHANOREQUAL64`
-geq(NumExpr_X,NumExpr_Y)                | `[NumExpr_X] [NumExpr_Y] GREATERTHANOREQUAL64`
+num64_eq(NumExpr_X,NumExpr_Y)             | `[NumExpr_X] [NumExpr_Y] EQUAL64`
+lt(NumExpr_X,NumExpr_Y)                 | `[NumExpr_X] [NumExpr_Y] LESSTHAN64`
+gt(NumExpr_X,NumExpr_Y)                 | `[NumExpr_X] [NumExpr_Y] GREATERTHAN64`
```
Comment (apoelstra) [inline doc/extension_spec.md:49]: In 2a08e90c3f8499399ce8614fd723969979ea6b8e:

Missing the `num64_` prefix. (The code is right, only the spec is wrong).

**Comment by apoelstra** [comment]
This looks good to me, though heads up it's a little weird to have these meaningful commits inside this PR.

**Review by apoelstra — APPROVED** [review]
ACK 85ed5f3d25740d48f0b30cabc2d04e354a3af585

**Comment by sanket1729** [comment]
@apoelstra, sorry about that. I had been using this branch to fix bugs that I catch in elements-miniscript while implementing the options project.

**Pull request merged**